src/main/resources/page/js/navbar-toggler-button-maker.js

```diff
@@ -0,0 +1,27 @@
+class NavBarTogglerButton {
+    constructor(){}
+
+    make(){
+        const button = this.addAttribute();
```

Comment:
Sometimes you use "const", "let", "nothing" for variable declaration.
Could you please choose one and use it everywhere, where that is possible?